README.md

```diff
@@ -15,12 +15,14 @@ QUICKINSTALL:
 
 ```
 # cd espressopp
-# cmake .
-# make
+# cmake -DEXTERNAL_BOOST=OFF -DEXTERNAL_MPI4PY=OFF .

```

Comment:
Recommending this as the default is really a bad idea:
https://travis-ci.org/junghans/espressopp/jobs/182439306

On the lastest ubuntu our internal boost is too old for the shipped gcc.